src/streaming/controllers/ProtectionController.js

```diff
@@ -66,17 +66,6 @@ MediaPlayer.dependencies.ProtectionController = function () {
             var self = this;
 
             // Build our request object for requestKeySystemAccess
-            var audioCapabilities = [], videoCapabilities = [];
-            if (videoInfo) {
-                videoCapabilities.push(new MediaPlayer.vo.protection.MediaCapability(videoInfo.codec));
-            }
-            if (audioInfo) {
-                audioCapabilities.push(new MediaPlayer.vo.protection.MediaCapability(audioInfo.codec));
-            }
-            var ksConfig = new MediaPlayer.vo.protection.KeySystemConfiguration(
-                    audioCapabilities, videoCapabilities, "optional",
-                    (self.sessionType === "temporary") ? "optional" : "required",
-                    [self.sessionType]);
             var requestedKeySystems = [];
```

Comment:
@bwidtmann The ProtectionModel_xxx classes are meant to handle differences in EME implementations between various browsers.  I need to look at the rest of the changes some more, but I'm pretty sure we need to add some special handling to the ProtectionModel_21Jan2015 class for this case.  If this is the only difference in EME API support between Firefox and Chrome, I think we can just add the special case instead of creating a whole new ProtectionModel_xxx class.  I'll provide you some separate guidance on how we should do that.